datacenter/ucp/2.2/guides/admin/backups-and-disaster-recovery.md

```diff
@@ -11,14 +11,39 @@ setting, you should
 
 The next step is creating a backup policy and disaster recovery plan.
 
+## Data managed by UCP
+
+UCP maintains data about:
+
+|         Data          |                                                  Description                                                  |
+| :-------------------- | :------------------------------------------------------------------------------------------------------------ |
+| Configurations        | The UCP cluster configurations                                                                                |
+| Access control        | Permissions for teams to swarm resources                                                                      |
```

Comment:
We also backup users, teams, and orgs. @wsong now that the UCP settings are managed as a config, does `docker/ucp backup` include data about the settings too, or information about the settings will only be backed-up if users to a swarm backup?